web/themes/custom/pineapple/components/_card.twig

```diff
@@ -24,7 +24,13 @@
         {% endfor %}
       </div>
     {% endif %}
-    <h2><a href="{{ url }}" rel="bookmark">{{ title }}</a></h2>
+    {% set title_link %}
+      <a href="{{ url }}" rel="bookmark">{{ title }}</a>
+    {% endset %}
```

Comment:
There's a better way to do this, because you may have other headings you need to be links, inside your heading component you can include the... Hmmm I've just realised. It'll probably be better if you had a separate pattern for a link component (that means not reusing the button component) and then inside of the heading component if you've got a heading_url (new variable you should create), you include the link pattern and pass the appropriate variables into it. so once it's all done the result mark up would look something like `<h3><a href="#">Cool title</a><h3>`

Reply:
So there are 2 variables inside the h tags like so:
```
<h1 {{ heading_attributes }}>{{ heading_url }}{{ heading_text }}</h1>
```
?